package.json

```
@@ -4,23 +4,20 @@
   "private": true,
   "description": "Provide scrapers for all major Israeli banks and credit card companies",
   "engines": {
-    "node": ">= 16.19.0",
-    "npm": ">= 6.14.8"
+    "node": ">= 18.19.0"
   },
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
     "clean": "rimraf lib",
     "test": "cross-env BABEL_ENV=test jest",
     "test:ci": "ncp src/tests/.tests-config.tpl.js src/tests/.tests-config.js && npm run test",
-    "lint": "eslint src --ext .ts",
+    "lint": "eslint src --ext .ts --fix",
```

I don't like to run the lint command and get it auto-fixed without my decision.

```suggestion
    "lint": "eslint src --ext .ts --fix",
    "lint:fix": "npm run lint -- --fix",
```